src/utils.js

```diff
@@ -166,3 +166,15 @@ export function readResAsString(response, autoDetectCharset) {
 			reader.readAsText(file, charset);
 		}));
 }
+
+export function evalCode(scriptSrc, code, evalCache = {}) {
+	const key = scriptSrc + code.length;
```

Comment:
Does `code.length` necessary? In which case the scriptSrc can not be the unique key of script?

Reply:
When you modify the code in `beforeExec` function,  should not use cache.But I think use `rawCode` as key is a waste of memory, so use `code.length` can meet most needs and use lower memory, what do you think about it?

Reply:
Not necessary to consider beforeExec modification so far, scriptSrc as key is enough.